src/Model/Parser.cs

```diff
@@ -1,28 +1,28 @@
 using System;
 using System.Collections.Generic;
 using System.Globalization;
-using System.Linq;
+using System.Text.RegularExpressions;
 using FuncSharp;
 
 namespace Mews.Fiscalization.SignatureChecker.Model
 {
     internal static class Parser
     {
         private static readonly CultureInfo FrenchCulture = new CultureInfo("fr-FR");
+        private static readonly Regex AmountRegex = new Regex(@"([-−]?[\d\s\\.,]*[\d\\.,])\s*([^\d\s\.]+)$");
 
         public static ITry<decimal, IEnumerable<string>> ParseDecimal(string value)
         {
-            var isSuccess = Decimal.TryParse(value.Replace('.', ',').Trim(), NumberStyles.Number, FrenchCulture, out var result);
+            var isSuccess = Decimal.TryParse(value.Replace('.', ',').Replace('−', '-').Trim(), NumberStyles.Number, FrenchCulture, out var result);
```

Comment:
I guess the `proper` solution is to serialize the amount and also the culture that was used, right?  Cause we are starting to reinvent the wheel here :D 
But it's too late now anyway, cause we need to support the old amounts as well.

Reply:
We need to support both cultures. Plus even the French culture doesn't work with the unicode minus sign character that Mews uses when formatting amounts in French culture.